django/views/i18n.py

```diff
@@ -52,14 +52,14 @@ def set_language(request):
                 if next_trans != next:
                     response = http.HttpResponseRedirect(next_trans)
             if hasattr(request, 'session'):
+                # Storing language in sessions is deprecated and will be removed in Django 2.1
```

Comment:
`# Storing language in sessions is deprecated - RemovedInDjango21Warning`
(for greppability when doing the removal)